frontend/angular.json

```diff
@@ -46,7 +46,10 @@
                         "tsConfig": "tsconfig.app.json",
                         "inlineStyleLanguage": "sass",
                         "assets": ["src/favicon.ico", "src/assets"],
-                        "styles": ["src/styles/index.sass"],
+                        "styles": [
+                            "src/styles/index.sass",
+                            "src/assets/fonts/fonts-montserrat.sass"
```

Comment:
I'm not sure but can font styling be import at index.sass, because I already have this problem in my task

Reply:
I'll add an import to index.sass and remove it from angular.json.